src/viur/core/bones/relational.py

```diff
@@ -46,6 +46,10 @@ class RelationalUpdateLevel(enum.Enum):
     """Update the relational information only when a new value is assigned to the bone."""
 
 
+class RelDict(t.TypedDict):
+    dest: "SkeletonInstance"
+    rel: "RelSkel" | None
```

Comment:
`str | ...` does not work, see https://github.com/viur-framework/viur-core/actions/runs/11370110093/job/31629115749?pr=1081
```suggestion
    rel: t.Optional["RelSkel"]
```